ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py

```diff
@@ -55,11 +55,11 @@ def generate_launch_description():
                         output='screen')
 
 
-    diff_drive_spawner = Node(
-        package="controller_manager",
-        executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
-    )
+    # diff_drive_spawner = Node(
```

Comment:
should we just delete this?  Maybe add a comment on why we'll keep it commented out if we want to keep it.